src/components/ContentHighlights/data/hooks.js

```diff
@@ -67,8 +67,10 @@ export function useHighlightSet(highlightSetUUID) {
  */
 export function useContentHighlightsContext() {
   const setState = useContextSelector(ContentHighlightsContext, v => v[1]);
+  const getState = useContextSelector(ContentHighlightsContext, v => v[0]);
+  const getHighlightsState = useCallback(() => getState(), [getState]);
```

Comment:
[clarification] Why is the context returning `getHighlightsState` when:
1. It doesn't appear to be called by any consumers of this context?
2. If anything were to call `getHighlightsState`, it wouldn't be taking advantage of the context selector via `useContextSelector` to extract the specific state you're looking for. The `useContextSelector` is essentially memoizing the state value(s) you're extracting, so by essentially returning _all_ state instead of just the slice of the state you want, you're not taking advantage of the performance optimizations from `useContextSelector` AFAIK.